setup.py

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/python3
+# -*- coding: utf-8 -*-
+
+"""Setup script."""
+
+# Copyright (c) 2023 SUSE LLC
+#
+# This file is part of csp-billing-adapter-amazon.
+
+from setuptools import setup
+
+with open('README.md') as readme_file:
+    readme = readme_file.read()
+
+with open('requirements.txt') as req_file:
+    requirements = req_file.read().splitlines()
+
+with open('requirements-test.txt') as req_file:
+    test_requirements = req_file.read().splitlines()[2:]
+
+with open('requirements-dev.txt') as req_file:
+    dev_requirements = test_requirements + req_file.read().splitlines()[2:]
+
+
+setup(
+    name='csp-billing-adapter-amazon',
+    version='0.0.1',
+    description='TBD',
+    long_description=readme,
+    long_description_content_type='text/markdown',
+    author='SUSE',
+    author_email='public-cloud-dev@susecloud.net',
+    url='https://github.com/SUSE-Enceladus/csp-billing-adapter-amazon',
+    entry_points={
+        'csp_billing_adapter': [
+            'amazon = csp_billing_adapter_k8s.plugin'
+        ]
+    },
+    packages=['csp_billing_adapter_amazon'],
```

Comment:
Is the above correct? not csp_billing_adapter_amazon.plugin

Reply:
Typo, thanks.